Tests/UI/New UI/BillingForm/View/SecurityCodeViewTests.swift

```diff
@@ -44,82 +46,4 @@ class SecurityCodeViewTests: XCTestCase {
       XCTAssertEqual(view.codeInputView.textFieldView.textField.text, "1234")
     }
 
-  func testCodeValidationWhenTextIsEmpty() throws {
-    let text = ""
-    let isValid = view.validateSecurityCode(with: text)
-    XCTAssertFalse(isValid)
-  }
-
-  func testCodeValidationWhenTextIsWhiteSpace() throws {
-    let text = " "
-   let isValid = view.validateSecurityCode(with: text)
-    XCTAssertFalse(isValid)
-  }
-
-  func testCodeValidationWhenTextWithOneNumber() throws {
-    let text = "1"
-   let isValid = view.validateSecurityCode(with: text)
-    XCTAssertFalse(isValid)
-  }
-
-  func testCodeValidationWhenTextWithTwoNumbers() throws {
-    let text = "12"
-   let isValid = view.validateSecurityCode(with: text)
-    XCTAssertFalse(isValid)
-  }
-
-  func testCodeValidationWhenTextWithThreeNumbers() throws {
-    let text = "123"
-    let isValid = view.validateSecurityCode(with: text)
-    XCTAssertTrue(isValid)
-  }
-
-  func testCodeValidationWhenTextWithFourNumbers() throws {
-    let text = "1234"
-    let isValid = view.validateSecurityCode(with: text)
-    XCTAssertTrue(isValid)
-  }
-
-  func testCodeValidationWhenTextWithFiveNumbers() throws {
-    let text = "12345"
-   let isValid = view.validateSecurityCode(with: text)
-    XCTAssertFalse(isValid)
-  }
-
-  func testCodeValidationWhenTextWithEmoji() throws {
-    let text = "📱"
-   let isValid = view.validateSecurityCode(with: text)
-    XCTAssertFalse(isValid)
-  }
-
-  func testCodeValidationWhenTextWithString() throws {
-    let text = "Hello"
-   let isValid = view.validateSecurityCode(with: text)
-    XCTAssertFalse(isValid)
-  }
-
-  func testCodeValidationWhenTextWithSpecialCharacter() throws {
-    let text = "123$"
-   let isValid = view.validateSecurityCode(with: text)
-    XCTAssertFalse(isValid)
-  }
-
-  func testCodeValidationWhenTextWithDouble() throws {
-    let text = "123.45"
-   let isValid = view.validateSecurityCode(with: text)
-    XCTAssertFalse(isValid)
-  }
-
-  func testCodeValidationWhenTextWithNegativeNumber() throws {
-    let text = "-123"
-   let isValid = view.validateSecurityCode(with: text)
-    XCTAssertFalse(isValid)
-  }
-
-  func testCodeValidationWhenTextWithWhiteSpacedNumbers() throws {
-    let text = "1 2 3 4"
-    let isValid = view.validateSecurityCode(with: text)
-    XCTAssertTrue(isValid)
```

Comment:
👍

Reply:
@ehab-al-cko hope you won't mind I'll continue the chat from code review (can't respond on that comment and we can use a thread):

> Please follow the git flow. why you want to merge on another ticket for events? PIMOB-987-i-os-log-the-start-of-the-payment-journey-with-the-new-ui

The task it was merging into was small and had some minor SwiftLint fixes included that would have cluttered this PR. Alternative was to create conflict on this branch once that was merged since they would both potentially fix the same swiftlint issues. Or to add more code to review that had nothing with the goal of the PR.

I agree, PRs should be aimed on main development branch unless it uses work from another branch to build on. In which case the merge order is important.